src/main/kotlin/no/nav/klage/oppgave/domain/klage/Ankebehandling.kt

```diff
@@ -0,0 +1,95 @@
+package no.nav.klage.oppgave.domain.klage
+
+import no.nav.klage.kodeverk.Fagsystem
+import no.nav.klage.kodeverk.Type
+import no.nav.klage.kodeverk.Ytelse
+import no.nav.klage.kodeverk.hjemmel.Registreringshjemmel
+import no.nav.klage.kodeverk.hjemmel.RegistreringshjemmelConverter
+import no.nav.klage.oppgave.domain.Behandling
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+import javax.persistence.*
+
+@Entity
+@DiscriminatorValue("anke")
+class Ankebehandling(
+    @Column(name = "dato_vedtak_klageinstans")
+    val datoVedtakKlageinstans: LocalDate,
+    @Column(name = "avsender_enhet_klageinstans")
+    val avsenderEnhetKlageinstans: String,
+
+    @ElementCollection(targetClass = Registreringshjemmel::class, fetch = FetchType.EAGER)
+    @CollectionTable(
+        name = "ankebehandling_hjemmel",
+        schema = "klage",
+        joinColumns = [JoinColumn(name = "ankebehandling_id", referencedColumnName = "id", nullable = false)]
+    )
+    @Convert(converter = RegistreringshjemmelConverter::class)
+    @Column(name = "id")
+    val inputhjemler: MutableSet<Registreringshjemmel> = mutableSetOf(),
+
+//    Finn ut hvordan dette skal fungere i anker etter hvert
+//    @Column(name = "dato_behandling_avsluttet_av_saksbehandler")
+//    var avsluttetAvSaksbehandler: LocalDateTime? = null,
+    //Fins i noen tilfeller, men ikke alle.
+    @Column(name = "forrige_klagebehandling_id")
+    var klagebehandlingId: UUID? = null,
+
+    //Common properties between klage/anke
+    id: UUID = UUID.randomUUID(),
+    klager: Klager,
+    sakenGjelder: SakenGjelder,
+    ytelse: Ytelse,
+    type: Type,
+    kildeReferanse: String,
+    dvhReferanse: String? = null,
+    sakFagsystem: Fagsystem? = null,
+    sakFagsakId: String? = null,
+    //Umulig å vite innsendt-dato.
+    innsendt: LocalDate? = null,
+    //Settes automatisk i klage, må kunne justeres i anke. Bør også representeres i delbehandlinger. Må gjøres entydig i anke, hører antageligvis ikke hjemme i felles klasse.
+    mottattKlageinstans: LocalDateTime,
```

Comment:
Felt som dette er jeg usikker på, for det kan være felles, og det betyr jo bare en ting, men den tingen er på en måte i ulike faser i saksbehandlingen for klager og anker.  For en anke så vil mottattKlageinstans == mottattNav, mens for en klage er det ikke det.

Reply:
Jeg tenker at det er ganske entydig i anketilfellet, og at vi kanskje kan ta en ny runde på hva vi kaller det i klage.

Reply:
Jeg er enig i at det er entydig i anketilfellet. Det er egentlig også entydig i klagetilfellet. Det jeg sliter med er bare at det i de to tilfellene ikke betyr samme ting. Men om det er et problem eller ikke klarer jeg ikke å bli enig med meg selv om..